tests/dragonfly/connection_test.py

```diff
@@ -0,0 +1,176 @@
+import asyncio
+import aioredis
+import async_timeout
+#from conftest import DATABASE_INDEX
+
+
+def verify_response(monitor_response: dict, key: str, value: str) -> bool:
```

Comment:
does it run with pytests? I am asking because you do not import pytests, not you use fixtures so what guarantees what dragonfly is up?

Reply:
Yes - it is running with pytest. The fixture is for the connection - see the file conftest.py where you have async_pool fixture. This fixture is depending on the running of the dragonfly instance. You normally don't need to "include" pytest directly to use it, pytest just discover the files under test and run them

Reply:
good to know. looks like magic.

Reply:
The pytest is a framework to running tests, the way you write them is very "free style" you can do whatever you like there - its not like a "normal" framework in C++ or other languages.

Reply:
regarding the link about the async fixture - you have to write it yourself, I tried it, and from some reason, the tests didn't work well after it (they sometimes pass, and sometimes didn't - like the messages didn't come in the right order).

Reply:
ok